src/features/map/map.selectors.ts

```diff
@@ -18,8 +18,8 @@ export const selectGeneratorConfigWithData = createSelector(
   ],
   (dataviews, tracks, events, highlightedTime, rulers, highlightedEvent) => {
     const generatorConfigsWithData = dataviews.map((dataviewWorkspace: DataviewWorkspace) => {
-      if (!dataviewWorkspace.dataview || !dataviewWorkspace.dataview.config) return null
-      const generatorConfig: Generators.GeneratorConfig = dataviewWorkspace.dataview.config
+      const generatorConfig: Generators.GeneratorConfig =
```

Comment:
why was these need before and it isn't anymore?

Reply:
TS was complaining somehow.